db/team.go

```diff
@@ -121,6 +122,22 @@ func (t *team) Delete() error {
 	return tx.Commit()
 }
 
+func (t *team) Rename(name string) error {
+	_, err := psql.Update("teams").
+		Set("name", name).
+		Where(sq.Eq{
+			"id": t.id,
+		}).
+		RunWith(t.conn).
+		Exec()
+
+	if err == nil {
+		t.name = name
```

Comment:
Oh, I think this can be removed; the tests shouldn't need it anymore. I'd rather leave those fields to only ever be set on first fetch, and keep the objects stateless.

Reply:
Understood, I will remove it.

Reply:
(ignore this comment, just testing out the github api)